docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,527 @@
+
+If joining from the genesis block, the output of this command looks similar to:
+```
+{
+  "name": "channel1",
+  "url": "participation/v1/channels/channel1",
+  "status": "active",
+  "consensusRelation": "follower",
+  "height": 1
+}
+```
+
+Otherwise, if joining from the latest config block, the `status` is "onboarding". In that case, after the channel ledger has caught up to the specified config block, you can issue the same `osnadmin channel list` command to confirm the status changes to **active** and the ledger height has caught up. For example, if the total number of blocks on the channel ledger at the specified config block was 1151, the output would look similar to:
+
+```
+{
+  "name": "channel1",
+  "url": "participation/v1/channels/channel1",
+  "status": "active",
+  "consensusRelation": "follower",
+  "height": 1151
+}
+```
+
+Note that to pull the blocks to the channel ledger, even as a follower, this new orderer must belong to an organization that is part of the current channel configuration, otherwise it is unable to replicate any blocks to its ledger. Also, notice the `consensusRelation` status is still **follower** because the node is not part of the channel consenter set.
+
+The diagram shows the new ordering node that has been added as a follower:
+
+![create-channel.osnadmin4](./osnadmin4.png)
+_OSN4 is added to the channel as a follower. Its status is "onboarding" while its ledger catches up to the config block provided on the osnadmin channel join command._
+
+
+While the orderer status is now **active**, it is still not able to participate in the ordering service because its `consensusRelation` status is still a **follower**, it will continue to pull blocks and stay current with the ledger. Because the status is now active, when you are ready for the orderer to transition from a follower to a consenter, you need to add the orderer to the consenter set on the channel by submitting a [channel configuration update transaction](../config_update.html), or by using the [fabric-config](https://github.com/hyperledger/fabric-config) library.  As a reminder,  you should never attempt any configuration changes to the Raft consenters, such as adding a consenter, unless all consenters are online and healthy as you risk losing quorum and your ordering service will not be able to process transactions.
+
+Ensure that the orderer organization has write privileges in the channel configuration so the orderer can become a consenter. When the channel update transaction is complete, you can use the `osnadmin channel list` command again to confirm that the `consensusRelation` status for the orderer automatically changes to **consenter**.
+
+```
+{
+  "name": "channel1",
+  "url": "participation/v1/channels/channel1",
+  "status": "active",
+  "consensusRelation": "consenter",
+  "height": 1151
```

Comment:
For correctness here, I'd bump the height to 1152 as the config update to add the consenter to the consenters set results in a block on the ledger.

Reply:
Done.